_pkgdown.yml

```diff
@@ -62,108 +62,114 @@ figures:
+        contents:
+          - contains("_opts")
+          - opts_list
+      - title: Preprocess data
+        desc: Functions used for prepropcessing data
+        contents:
+          - fill_missing
+          - add_breakpoints
+          - filter_leading_zeros
+          - apply_zero_threshold
+      - title: Summarise Across Regions
+        desc: Functions used for summarising across regions (designed for use with regional_epinow)
+        contents:
+          - regional_summary
+          - regional_runtimes
+          - get_regional_results
+      - title: Summarise results
+        desc: Functions for summarising results
+        contents:
+          - contains("summary.")
+          - contains("calc_")
+          - make_conf
+          - map_prob_change
+      - title: Plot Results
+        desc: Plot generated results
+        contents:
+          - contains("plot")
+      - title: Report results
+        desc: Functions to report results
+        contents:
+          - report_plots
+          - report_summary
+      - title: Define and Parameterise Distributions
+        desc: Functions to define and parameterise distributions
+        contents:
+          - Distributions
+          - contains("dist_spec")
+          - bound_dist
+          - collapse
+          - discretise
+          - fix_parameters
+          - get_parameters
+          - get_pmf
+          - get_distribution
+          - is_constrained
+      - title: Fit delay Distributions
+        desc: Functions to fit delay distributions
+        contents:
+          - bootstrapped_dist_fit
+          - dist_fit
+      - title: Simulate
+        desc: Functions to help with simulating data or mapping to reported cases
+        contents:
+          - simulate_infections
+          - simulate_secondary
+          - convolve_and_scale
+      - title: Data
+        desc: Package datasets that may be used to parameterise other functions or in examples
+        contents:
+          - example_generation_time
+          - example_incubation_period
+          - example_reporting_delay
+          - example_confirmed
+          - example_truncated
+      - title: Get/Extract Data
+        desc: Functions for extracting data from objects or getting data from sources
+        contents:
+          - contains("get_")
+          - contains("extract_")
+      - title: Clean Data
+        desc: Functions for cleaning data
+        contents:
+          - contains("clean")
+      - title: Setup Utilities
+        desc: Function used for setting up functionality
+        contents:
+          - contains("setup_")
+      - title: Utilities
+        desc: Utility functions
+        contents:
+          - run_region
+          - expose_stan_fns
+          - convert_to_logmean
+          - convert_to_logsd
+          - growth_to_R
+          - R_to_growth
+          - update_secondary_args
+      - title: internal
+        contents:
+          - has_keyword("internal")
+  - title: Stan Reference
```

Comment:
The copy and paste here has confused me but I think the only change is adding the link to the stan docs? I think rather than cluttering the top of the pkgdown packagte we may want to keep the Reference top level and have a drop down for R and stan? That would be nice as well at it matches what we do in primarycensored

Reply:
I've made it a dropdown in eaef2a16cf93d4200233d804568d275d98322274.